File: Algoritmos de ordenamiento/JavaScript/BubbleSort.js
function bubbleSort(array) {
  let length = array.length;
  for (let i = 0; i < length; i++) {
    for (let j = 0; j < length - i - 1; j++) {
      if (array[j] > array[j + 1]) {
        let num = array[j];
        array[j] = array[j + 1];
        array[j + 1] = num;
      }
    }
  }
}

let array = [5, 4, 3, 2, 1];
bubbleSort(array);

console.log(array);

File: Algoritmos de ordenamiento/Java/BubbleSort.java
package Java;

public class BubbleSort {
    public static void bubbleSort(int[] array) {
        int length = array.length;
        for (int i = 0; i < length; i++) {
            for (int j = 0; j < length - i - 1; j++) {
                if (array[j] > array[j + 1]) {
                    int num = array[j];
                    array[j] = array[j + 1];
                    array[j + 1] = num;
                }
            }
        }
    }
}
